_studio/mfx_lib/vpp/include/mfx_vpp_defs.h

```diff
@@ -157,6 +158,575 @@ typedef enum
     MIRROR_WO_EXEC = 0x0004
 } MirroringPositions;
 
+typedef enum
+{
```

Comment:
also, I believe you need to add a comment before declaration of this enums, that order here matters. So no one would change the order of enum entries